task_8/task_8_2.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("node:fs/promises");
+const path = require("node:path");
+
+const filePath = path.join(__dirname, "file.txt");
+async function changePermissions(filePath) {
+  try {
+    await fs.chmod(filePath, 0o400);
```

Comment:
Let's use fs.constants for this permission code.

Reply:
Fixed!